src/js/components/Menu/Menu.js

```diff
@@ -275,7 +281,9 @@ const Menu = forwardRef((props, ref) => {
             onEnter={onSelectMenuItem}
           >
             <ContainerBox background={dropBackground || theme.menu.background}>
-              {align.top === 'top' ? controlMirror : undefined}
+              {menuMirrorReference === 'top' && align.top === 'top'
```

Comment:
Thinking functionality looks great. Just wondering if this variable could have a more clear name.

maybe, `alignControlMirror` and `setAlignControlMirror` ?

Reply:
I'm open to any suggestions. I must confess I spent a couple minutes renaming and trying new names before committing so it would be more clear haha. I'll be pushing a new commit to remove a comment that wasn't supposed to be there and guess  we'll go with `alignControlMirror` and `setAlignControlMirror`. :smile: